sos/plugins/openstack_tripleo.py

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2019 Red Hat, Inc., Cedric Jeanneret <cjeanner@redhat.com>
+
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import os
+
+
+class OpenStackTripleO(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    '''Some TripleO logs and files (OpenStack installer)
+    Recursion level is max 2 for both listed directories.
+    Those files are useful in order to debug a failed install, since the
+    whole deploy is based on them. We find those directories on all the nodes.
+    '''
+    plugin_name = 'openstack_tripleo'
+
+    def setup(self):
+        self.add_copy_spec('/var/log/paunch.log')
+        self.add_copy_spec('/var/lib/container-puppet')
+        self.add_copy_spec('/var/lib/tripleo-config')
```

Comment:
Is there any sensitive information in this config file? Or `container-puppet` for that matter?

Reply:
The directory has files with passwords.

Reply:
hm, right - I think I've seen something about masking password in another plugin. Lemme check that and see if I can do something clean about that. Will deploy a thing on my side in order to inspect that.